docs/codesigning/common-issues.md

```diff
@@ -32,8 +32,121 @@ Your project defines a provisioning profile which doesn't exist on your local ma
 
 Make sure to have a valid code signing identity defined in your project targets. This might happen when you select `Don't Code Sign` as Code Signing Identity.
 
-## fastlane errors
+## _fastlane_ errors
 
 > Could not find a matching code signing identity for type 'X'
 
 There are no certificates available on the Apple Developer Portal. This could either mean someone revoked a certificate, or you don't have access to it. 
+
+# Debugging Steps for "No applicable devices found." error
+
+It's not immediately obvious what this error is or what it's caused by, so hopefully these simple steps will help you find the root cause faster than my experience with it.
+
+### Not Just _fastlane_
+
+This error happens to all projects built with `xcodebuild` / `xcrun`, so if you're trying to blame _fastlane_ here, take a deep breath and look elsewhere first. _fastlane_ might help alleviate it with some env var unsetting / setting or other tricks (see below), but this is an error from the new export APIs that Xcode uses.
```

Comment:
Let's replace `so if you're trying to blame _fastlane_ here, take a deep breath and look elsewhere first` with `and it not caused by _fastlane_`